analyzer/pkg/rpmcmd/rpm.go

```diff
@@ -115,7 +114,7 @@ func outputPkgInfo(dir string) (out []byte, err error) {
 	const newFmt = "%{NAME} %{EPOCHNUM} %{VERSION} %{RELEASE} %{SOURCERPM} %{ARCH}\n"
 	out, err = exec.Command("rpm", "--dbpath", dir, "-qa", "--qf", newFmt).Output()
 	if err != nil {
-		return exec.Command("rpm", "--dbpath", dir, "-qa", "--qf", oldFmt).Output()
+		return out, err
```

Comment:
Would you stay this line? `oldFmt` is used here.

Reply:
Added back in.

Reply:
Thanks!